tests/lib/test_Repo.py

```diff
@@ -6,16 +6,27 @@
 import pytest
 from onyo import Repo, OnyoInvalidRepoError
 
+def params(d):
```

Comment:
This should be added to `conftest.py` and registered as a fixture. That way it doesn't need to be added to each of the test files.